File: buff/classes/354/pub/etc/student.gccgo/person.go
package person

import . "fmt"

type IPerson interface {
	Name() string
	String() string
}

type Person struct {
	name string
	age int
}

func NewPerson(name string, age int) (p *Person) {
	p=new(Person)
	p.Init(name,age)
	return
}

func (p *Person) Init(name string, age int) {
	p.name=name
	p.age=age
}

func (p *Person) Name() string { return p.name }

func (p *Person) String() string {
	return p.name+" "+Sprintf("%d",p.age)
}

File: buff/classes/354/pub/etc/Student/Person.java
public class Person {

    private String name;
    private int age;

    public Person(String name, int age) {
        init(name,age);
    }

    private void init(String name, int age) {
        this.name=name;
        this.age=age;
    }

    public String name() { return name; }

    protected String string() { return name()+" "+age; }

    public String toString() { return string(); }

}
